lib/stm32wb0/README

```diff
@@ -0,0 +1,123 @@
+      Middlewares/ST/STM32_BLE/stack/include/ble_gatt.h
+      Middlewares/ST/STM32_BLE/stack/include/uuid.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_stack.h
+      Middlewares/ST/STM32_BLE/ble.h
+      Middlewares/ST/STM32_BLE/stack/include/bleplat_cntr.h
+      Middlewares/ST/STM32_BLE/stack/include/bleplat.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_const.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_status.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_stack_user_cfg.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_api.h
+      Middlewares/ST/STM32_BLE/stack/include/ble_events.h
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/dm_alloc.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/dtm_cmd_db.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/aci_adv_nwk.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/Target/bleplat_cntr.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/Target/bleplat.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Modules/PKAMGR/Src/pka_manager.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Interfaces/hw_aes.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Interfaces/hw_pka.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Interfaces/hw_rng.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Modules/blue_unit_conversion.s
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Modules/osal_memcpy.s
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/adv_buff_alloc_tiny.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/transport_layer.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/dtm_preprocess_events.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/adv_buff_alloc.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Modules/RADIO_utils/Src/RADIO_utils.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/System/Modules/miscutil.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/pawr_buff_alloc.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/STM32_BLE/App/dtm_cmds.c
+      Projects/NUCLEO-WB09KE/Applications/BLE/BLE_TransparentMode/Core/Src/stm32wb0x_hal_msp.c
+      Middlewares/ST/STM32_BLE/stack/config/ble_stack_user_cfg.c
+
+Dependencies:
+   This library depends on STM32Cube HAL API.
+   It is available in stm32cube/stm32wb0x/drivers
+
+URL:
+   https://github.com/STMicroelectronics/STM32CubeWB0
+
+Commit:
+   61d80e00cdb6136a58a33b95d2137e3bc9baa28e
+
+Maintained-by:
+   Internal
+
+License:
+   ULTIMATE LIBERTY SOFTWARE LICENSE AGREEMENT
```

Comment:
License of files included this module should be OSI approved (BSD-3 for instance)

Reply:
Fixed.